android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -511,6 +516,21 @@ protected void onSafeCreate(@Nullable Bundle savedInstanceState)
      */
     if (Map.isEngineCreated())
       onRenderingInitializationFinished();
+
+    // Setup BroadcastReceiver to receive changes of system locale.
+    if (mLocaleChangeReceiver == null)
```

Comment:
Is onSafeCreate called only once per lifetime of the activity? If yes, then is this check really needed?

Reply:
Yes, `onSafeCreate()` is only called once for the activity, so I've removed the check...